packages/mock-account-provider/src/apolloClient.ts

```diff
@@ -0,0 +1,59 @@
+import type { NormalizedCacheObject } from '@apollo/client'
+import {
+  createHttpLink,
+  ApolloLink,
+  ApolloClient,
+  InMemoryCache
+} from '@apollo/client'
+import { setContext } from '@apollo/client/link/context'
+import { CONFIG } from './parse_config'
+import { onError } from '@apollo/client/link/error'
+import fetch from 'cross-fetch'
+
+const httpLink = createHttpLink({
+  uri: CONFIG.self.graphqlUrl,
+  fetch
```

Comment:
When I was trying this before i was using a stale built file (stale `run_seed.js`). When I switched to using this one it complained that the Link here needed a supplied `fetch`. This can probably be removed once within remix